code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1954,17 +1948,25 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		// Display alerts based on the amount of cold damage being taken
 		// Apply more damage based on how cold you are
 
-		if(body_temp < 120)
+		if(body_temp < bodytemp_cold_damage_limit - 15)
 			H.throw_alert("temp", /atom/movable/screen/alert/shiver, 3)
-			H.apply_damage(COLD_DAMAGE_LEVEL_3 * coldmod * H.physiology.cold_mod, BURN)
+			if(H.stat != DEAD) // probably can store them in cold storage like this
+				H.apply_damage(COLD_DAMAGE_LEVEL_3 * coldmod * H.physiology.cold_mod, BURN)
+			H.emote("shiver")
 
-		else if(body_temp < 200)
+		else if(body_temp < bodytemp_cold_damage_limit - 7)
 			H.throw_alert("temp", /atom/movable/screen/alert/shiver, 2)
-			H.apply_damage(COLD_DAMAGE_LEVEL_2 * coldmod * H.physiology.cold_mod, BURN)
+			if(H.stat != DEAD) // when you think about it, being cold wouldnt do skin damaage if there nothing even alive?
```

Comment:
I mean if you had no water in your cells mabye, but unless your a tardigrade you would take damage from extreme colds

Reply:
I do think there's an argument to be made about like. once all the water has frozen. there's not really more damage to be done

Either way minimizing things that continue to tick on corpses is probably good

Reply:
Unless you are decently far from a star (around jupiter or greater i think), the risk is actually from burning up from solar radiation.

After that, you will freeze through.... After several days.

There will be a rapid drop initially from water at or near the surface of skin vaporizing and taking heat, and some risk with embolisms and popped lungs, but once a heart ceases, damage progresses far more slowly.